lnbits/settings.py

```diff
@@ -245,6 +245,13 @@ def from_dict(cls, d: dict):
             **{k: v for k, v in d.items() if k in inspect.signature(cls).parameters}
         )
 
+    # fixes openapi.json validation, remove field env_names
```

Comment:
- is there no side effect from removing `env_names`?
- when was `env_names` added and why?

Reply:
`env_names` is added by BaseSettings class https://github.com/pydantic/pydantic/blob/3703caa3725a2d9190d37f7af7aab0fa2d096fae/pydantic/v1/env_settings.py#L118

it seems to me its just additional metadata, and not uses anywhere else. i did not notice anything testing.

Reply:
got it!